pkg/nsx/services/subnet/subnet.go

```diff
@@ -180,6 +180,30 @@ func (service *SubnetService) DeleteSubnet(nsxSubnet model.VpcSubnet) error {
 	return nil
 }
 
+func (service *SubnetService) ListSubnet() []model.VpcSubnet {
+	subnets := service.SubnetStore.List()
+	subnetSet := []model.VpcSubnet{}
```

Comment:
can we use a different variable name instead of "subnetSet"? it is confusing.

Reply:
done

Reply:
still subnetSet?